### PR TITLE
golib: fix import path for thrift

### PR DESCRIPTION
**Pull request by jgheewala**

### CODE REVIEW

File: sfxclient/httpsink.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/signalfx/golib/v3/sfxclient/spanfilter"
 	"github.com/signalfx/golib/v3/trace"
 	traceformat "github.com/signalfx/golib/v3/trace/format"
-	"github.com/signalfx/sapm-proto/translator"
+	"github.com/signalfx/golib/v3/translator"
```
Comment (owais) [inline sfxclient/httpsink.go]: Perhaps should should be placed under the trace package? So the import path looks like `"github.com/signalfx/golib/v3/trace/translator/sfxtosapm"`

Comment (jgheewala) [reply sfxclient/httpsink.go]: yup much better thanks

Comment (jgheewala) [reply sfxclient/httpsink.go]: done

**Comment by owais** [comment]
LGTM but perhaps should wait for another +1

**Pull request merged**